src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -147,6 +147,12 @@ const resolvePage = async(
 ) => {
   const {dispatch, getState} = services;
   const state = getState();
+
+  if (match.params.book && 'slug' in match.params.book && !match.params.page) {
+    // book slug provided but no page
+    await processBrowserRedirect(services);
```

Comment:
if you allow this to hit the page not found because it has the wrong slug conditional [below](https://github.com/openstax/rex-web/pull/2180/files#diff-3b681369c849d5f3e705f6e56269f0a1c93047c1049cd36b99f3e9335ae62db4R158) it'll get redirected through the existing logic

Reply:
but the logic below requires a page id and we don't have that